packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -67,12 +67,14 @@ const PatientSearchLaunch: React.FC<PatientSearchLaunchProps> = () => {
   const { t } = useTranslation();
   const layout = useLayoutType();
   const [open, setOpen] = useState<boolean>(false);
+  const [showResultsPanel, setShowResultsPanel] = useState<boolean>(false);
```

Comment:
It took me a minute to figure out what's going on here. Let's rename `open`/`setOpen` to `showSearchInput`/`setShowSearchInput` (or something like that). With your change, simply "open" is now ambiguous.